<!DOCTYPE HTML PUBLIC "-//W3C//DTD HTML 4.01 Transitional//EN"
  "http://www.w3.org/TR/html4/loose.dtd">
<html>
  <head>
    <title>$escape(traceback.exception) // Werkzeug Debugger</title>
    <link rel="stylesheet" href="./__debugger__?resource=style.css" type="text/css">
    <script type="text/javascript" src="./__debugger__?resource=jquery.js"></script>
    <script type="text/javascript" src="./__debugger__?resource=debugger.js"></script>
  </head>
  <body>
    <div class="debugger">
      <h1>$escape(traceback.exception_type)</h1>
      <div class="detail">
        <p class="errormsg">$escape(traceback.exception)</p>
        <p class="location">
          <cite>"$escape(traceback.frames[-1].filename)"</cite>, line
          <em>$traceback.frames[-1].lineno</em>, in
          <code>$escape(traceback.frames[-1].function_name)</code>
        </p>
      </div>
      <div class="explanation">
        The debugger cought an exception in your WSGI application.  You can now
        look at the traceback which lead to the error.  If you have JavaScript
        enabled you can also click on the buttons that appear on mouse-over to
        open a python shell for that frame or view the sourcecode of that module.
      </div>
      <h2>Traceback <em>(most recent call last)</em></h2>
      $traceback.render_summary(include_title=False)
      <div class="footer">
        Brought to you by <strong class="arthur">DON'T PANIC</strong>, your
        friendly Werkzeug powered traceback interpreter.
      </div>
    </div>
  </body>
</html>
